src/tests/ftest/deployment/agent_failure.yaml

```diff
@@ -0,0 +1,42 @@
+hosts:
+    test_servers:
+        - server-A
+    test_clients:
+        - client-A
+
+timeout: 300
+
+setup:
+    start_agents_once: False
+
+server_config:
+    name: daos_server
+    servers:
```

Comment:
We should use 2 engines per server node configuration.

Reply:
Also, does this need to run on HW Large in CI or could we use HW Medium with 3 servers (6 engines) and 1 client?

Reply:
Done. Now we're creating two engines per node.
It's running on HW Large, so we're using 7 servers, 14 engines.

Reply:
> Also, does this need to run on HW Large in CI or could we use HW Medium with 3 servers (6 engines) and 1 client?

I am fine either way. I understand from resources point of view it will make sense to have it on HW medium though.